pkg/runtime/bindparam.go

```diff
@@ -35,7 +33,7 @@ func BindStyledParameter(style string, explode bool, paramName string,
 	value string, dest interface{}) error {
 
 	if value == "" {
-		return echo.NewHTTPError(http.StatusBadRequest, "parameter '%s' is empty, can't bind its value", paramName)
+		return fmt.Errorf("parameter '%s' is empty, can't bind its value", paramName)
```

Comment:
This change has one unfortunate side effect. Now, when invalid parameters are specified, what were formerly HTTP/400 errors will now be HTTP/500 errors. So, we need to either return some error wrapper with a status code which the Chi or Echo layer can turn into the relevant error, or all the call sites in codegen need to make this an HTTP/400.

Reply:
Looking at the generate code and the templates, it seems that in both codegen this error is already wrapped into a HTTP/400 error. See [wrappers.tmpl#L39](https://github.com/deepmap/oapi-codegen/blob/master/pkg/codegen/templates/wrappers.tmpl#L39) and [chi-middleware.tmpl#L75](https://github.com/deepmap/oapi-codegen/blob/master/pkg/codegen/templates/chi-middleware.tmpl#L75).

Reply:
You are correct. I didn't notice.